shared/wallets/send-form/asset-input/asset-input-advanced.tsx

```diff
@@ -396,6 +398,7 @@ const styles = Styles.styleSheetCreate(
       },
       error: {
         color: Styles.globalColors.redDark,
+        maxWidth: '80%',
```

Comment:
🤔There might be another way around this. I'll take a peek.

Reply:
lol